content/blog/2022/03/cri-o-cve-mitigation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: CRI-O CVE-2022-0811 Mitigation
+authors:
+- José Guilherme Vanz
+date: 2022-03-29
+---
+
+Recently a severe [CVE](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-0811)
+in the CRI-O container engine come to public. The flaw in CRI-O allows bad
+actors to gain root access and run arbitrary code in the host machine.
+
+A [fix](https://github.com/cri-o/cri-o/releases/tag/v1.23.2) for the issue is
+already available and you should update your cluster to avoid any headache in
+the future. But if you cannot do that now, you can use Kubewarden to mitigate
+this issue. It's possible to prevent pods with `sysctl` configuration
+to run in the cluster with the `sysctl-psp-policy` policy available in the [Policy
+Hub](https://hub.kubewarden.io/).
+
+The policy configuration to block all pod sysctl configuration is the following:
+
+```yaml
+apiVersion: policies.kubewarden.io/v1alpha2
+kind: ClusterAdmissionPolicy
+metadata:
+  name: mitigate-crio-cve
+spec:
+  policyServer: default
```

Comment:
`policyServer: default` can be removed, since default will be used if no `PolicyServer` is provided